### ISSUE TITLE
Bug: Plot of side force coefficient is constant at zero

### ISSUE THREAD

**Issue by rkuhlf** [openrocket/openrocket#1060]
Plots of the side force coefficient for any rocket simulation show it as being constant at zero, regardless of AOA.
![SideForce](https://user-images.githubusercontent.com/29072641/141222395-478f23d3-3c41-4db3-9cb6-4aa4cdd7273b.png)

**Comment by JoePfeiffer** [comment]
Yes, this is not calculated at present.  The various finsets have commented-out code similar to this (this is from TubeFinSetCalc.java):
/*
		 * TODO: HIGH:  Compute actual side force and yaw moment.
		 * This is not currently performed because it produces strange results for
		 * stable rockets that have two fins in the front part of the fuselage,
		 * where the rocket flies at an ever-increasing angle of attack.  This may
		 * be due to incorrect computation of pitch/yaw damping moments.
		 */
		//		if (fins == 1 || fins == 2) {
		//			forces.Cside = fins * cna1 * Math.cos(theta-angle) * Math.sin(theta-angle);
		//			forces.Cyaw = fins * forces.Cside * x / conditions.getRefLength();
		//		} else {
		//			forces.Cside = 0;
		//			forces.Cyaw = 0;
		//		}
		forces.setCside(0);
		forces.setCyaw(0);
